### PR TITLE
[ONNX] Do not reshape input to DequantizeLinear if scalar

### PR DESCRIPTION
**Pull request by postrational**
### Details:
DequantizeLinear decomposition from onnx importer fails for opset13 when scale/zero_point is a scalar. ONNX Spec says that scalar is supported and applied to the entire tensor, shouldn't be reshaped along channel dimension.

### Tickets:
 - *53007*

### CODE REVIEW

**Comment by tomdol** [comment]
I'm not sure but this could potentially affect the Q/DQ related transformations. I think that was tested and a test would fail if there's a shape expectation there but I'd like to ask @jane-intel for a quick look too.

File: ngraph/frontend/onnx_import/src/op/dequantize_linear.cpp [hunk 1]
```diff
@@ -128,8 +128,15 @@ namespace ngraph
                                                                 const int64_t axis,
                                                                 const PartialShape& x_shape)
                     {
-                        std::vector<int64_t> target_dims;
+                        auto input_rank = input.get_partial_shape().rank();
 
+                        // Do not reshape input, if it contains a scalar value
+                        if (input_rank.is_static() && input_rank.get_length() == 0)
+                        {
+                            return input.get_node_shared_ptr();
```
Comment (ilyachur) [inline ngraph/frontend/onnx_import/src/op/dequantize_linear.cpp:136]: Can we add tests to cover the new behavior?

**Comment by bsledz** [comment]
> I'm not sure but this could potentially affect the Q/DQ related transformations. I think that was tested and a test would fail if there's a shape expectation there but I'd like to ask @jane-intel for a quick look too.

@jane-intel Could you please take a look?

File: ngraph/frontend/onnx_import/src/op/dequantize_linear.cpp (hunk 1 above)
Comment (bsledz) [reply ngraph/frontend/onnx_import/src/op/dequantize_linear.cpp:136]: Sure, done

**Review by etusien — APPROVED** [review]

**Review by tomdol — APPROVED** [review]

**Review by mbencer — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Review by slyalin — APPROVED** [review]

**Review by tsocha — APPROVED** [review]

**Pull request merged**